src/main.rs

```diff
@@ -151,6 +151,29 @@ fn get_input_files(files: Vec<String>) -> anyhow::Result<Vec<String>> {
     return Ok(input.lines().map(|f| f.to_string()).collect());
 }
 
+fn get_input_files(files: Vec<String>) -> anyhow::Result<Vec<String>> {
+    let mut input_files = get_input(files)?;
+    // This is a special case where we want to expand `.` and `..`.
+    let dots = vec![".", ".."];
```

Comment:
note: I think you can use a simpler type here since it is a static list of elements.
With a slice of an array instead of a Vec:
```suggestion
    let dots = &[".", ".."];
```

Reply:
> That'd be neat, but I don't have any experience with CD in github.

Oh yeah wasn't trying to get you to do it, just noting the bad state of it right now. Thank you for the PR :pray: